### DISCUSSION TITLE
Update column filter dropdown

### DISCUSSION THREAD

**Discussion by hilayshah111** [KevinVandy/material-react-table#1160] [Q&A]
Hey, after I get the api response, the column filter dropdown is set on initialization. The Api response got updated in the next call, how do I update the column filter dropdown to include the new values after the api response?

**Comment by KevinVandy** [comment]
Just do it. Make sure useMemo does are correct

**Reply by hilayshah111** [reply]
I am getting the array from a parent component. Can you provide an example to use table.setColumnFilterFns and onColumnFilterFnsChange?
